src/rng/test/time_initkeyctr.h

```diff
@@ -29,8 +29,8 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
-#ifndef TIME_INITKEYCTR_H__
```

Comment:
I know the leading double underscore is used internally, does LLVM also warn (or throw an error) with the trailing double underscores?

Reply:
Yeah.  Newer versions of LLVM are very picky about this stuff. "violates C++ standard and may lead to undefined behavior" or similar warning.

Reply:
Great, thanks.